tests/integration/targets/azure_ops_test_azure_manage_postgresqlflex/tasks/create_and_delete_postgresql.yml

```diff
@@ -0,0 +1,123 @@
+---
+- name: Test Create and Delete PostgreSQL Server
+  block:
+    - name: Print test step details
+      ansible.builtin.debug:
+        msg: "Run test with resource_group={{ test_resource_group }}"
+
+    - name: Set vars for test
+      ansible.builtin.set_fact:
+        azure_manage_postgresqlflex_firewall_rules:
+          - name: "rule_0"
+            start_ip_address: "172.10.1.0"
+            end_ip_address: "172.10.128.0"
+        azure_manage_postgresqlflex_database_instances:
+          - name: "database-test"
+
+    # Test: Create PostgreSQL Server
+    - name: Create PostgreSQL Server
+      ansible.builtin.include_role:
+        name: cloud.azure_ops.azure_manage_postgresqlflex
+      vars:
+        azure_manage_postgresqlflex_operation: create
+        azure_manage_postgresqlflex_resource_group: "{{ test_resource_group }}"
+    
+    # Verify that PostgreSQL Server was created as expected
+    - name: Gather PostgreSQL Server info
+      azure.azcollection.azure_rm_postgresqlflexibleserver_info:
+        resource_group: "{{ test_resource_group }}"
+        name: "{{ azure_manage_postgresqlflex_name }}"
+      register: _postgresql
+
+    - name: Assert that PostgreSQL Server was created as expected
+      ansible.builtin.assert:
+        that:
+          - _postgresql.servers | length == 1
+          - _postgresql.servers[0].name == azure_manage_postgresqlflex_name
+          - _postgresql.servers[0].sku.name == azure_manage_postgresqlflex_sku.name
+          - _postgresql.servers[0].storage.storage_size_gb == azure_manage_postgresqlflex_storage.storage_size_gb
+          - _postgresql.servers[0].administrator_login == azure_manage_postgresqlflex_admin_username
+          - _postgresql.servers[0].version == "11"
```

Comment:
ensure the type of the version field either by `| int == 11` *or* `| string == "11"`